src/content/docs/dropins/product-details/pdp-initialization.mdx

```diff
@@ -0,0 +1,96 @@
+---
+
+import Diagram from '@components/Diagram.astro';
+import Callouts from '@components/Callouts.astro';
+import OptionsTable from '@components/OptionsTable.astro';
+import Aside from '@components/Aside.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+Customizing the PDP dropin initializer can help you meet your project requirements and use cases.
+
+## Configuration options
+
+The PDP dropin initializer allows you to define the language definitions, default locale, and models that your project will use.
+
+<OptionsTable
+  options={[
+    ['Option', 'Type', 'Req?', 'Description'],
+    ['langDefinitions', 'object', 'No', 'Provides language definitions for internationalization (i18n).'],
+    ['defaultLocale', 'string', 'No', 'Specifies the default locale. If not provided, defaults to en-US.'],
+    ['models', 'object', 'No', 'Defines the data structure and initial data. Options include initialData, transform, and fallBackData.'],
+  ]}
+/>
+
+:::note
+Models are used to define the data structure and initial data for the PDP dropin. The `initialData` object contains product details, options, and attributes.
+:::
+
+## Example
+
+The following code shows the default implementation of the PDP dropin initializer in this Commerce boilerplate:
+
+```typescript
+// Define the models object with a ProductDetails property
+const models = {
+ProductDetails: {
+  initialData: { ...product },
+  },
+};
+
+// Initialize Dropins
+initializers.register(productApi.initialize, {
+  langDefinitions,
+  models,
+});
+```
+
+## Set default product options
+
+When a user navigates to the product detail page (PDP) on your site, you can set certain options as preselected defaults. This use case allows merchandisers to set default options through the Adobe Commerce Admin, which provides a more customized and streamlined shopping experience.
+
+* **Simple products**—Default option selection is supported out-of-the-box through the Admin (`isDefault`). No additional PDP dropin customization is required.
+
+* **Complex products**—Default option selection is _not_ supported out-of-the-box. Instead, you must use [product attributes](https://experienceleague.adobe.com/en/docs/commerce-admin/catalog/product-attributes/product-attributes) and customize the PDP dropin initializers to define which attribute and value is used by default. If no option is defined as default, the first listed option will be preselected by default.
+
+This approach allows merchants to set different default selections for complex products in the Admin. Developers can then customize the PDP dropin to show the default or preselected options.
+
+<Steps>
+
+1. Log in to the Adobe Commerce Admin.
+
+1. Create a custom attribute (`default_options`) to define default options for each product.
+
+1. Ensure that the `default_options` attribute has been [exported](https://experienceleague.adobe.com/en/docs/commerce-merchant-services/saas-data-export/overview) to the Catalog Service.
```

Comment:
Ensure that it has been exported _**and**_ has visible in PDP role.

Reply:
Does it refer to [this](https://experienceleague.adobe.com/en/docs/commerce-admin/catalog/products/types/product-create-configurable#step-4-complete-the-basic-settings:~:text=Accept%20the%20default%20Visibility%20setting%20of%20Catalog%2C%20Search.) or something else?

Reply:
Ohhh, it must be `roles` in the [ProductViewAttribute](https://developer.adobe.com/commerce/services/graphql/catalog-service/products/#productviewattribute-type) type. I'm going to push a change, but please let me know if it's something else.